guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+- Static deposit addresses for exchanges: Users looking to deposit bitcoin at exchanges may get confused when they see changing/different deposit addresses. Exchanges can now share static addresses with users; this will keep deposit addresses the same while allowing the exchange to map deposit transactions to users
+
+
+### Contacts
+
+Contacts are another feature that are greatly improved with BIP-352. Since users can safely use the same silent payment address for multiple payments, it is natural for them to want to store these for future use. Contacts are a great way to store them in terms that users can intuit: names and faces/images. The Contacts page here provides good guidance about the topic.
+
+Labels help Contacts in another interesting way by enabling ‘sender contacts’ where the user can create Contacts for parties they only receive bitcoin from. This is accomplished by adding a different label for every contact.
+
+{% include image-gallery.html pages = page.images_contacts %}
+
+Overall, Silent payments present a great opportunity to start basing payments UX in terms of people instead of addresses, and applications.
+
+{% include tip/recommendation.html %}
+
+Wallets should create an in-app contact list and avoid storing bitcoin payment information in the operating system’s default contact list since it cannot guarantee its safety for data breaches or phone hacks.
+
+{% include tip/close.html %}
+
+
+## Setup
+
+Applications can take a variety of approaches to set up and communicate BIP-352 wallets. Since such a wallet has a reusable address, this could be introduced and explained to users during the setup process. Some wallets may use unique or significant locations (mobile widget, custom app logos, watch faces) to house this static address for easy retrieval, and may highlight the same during setup.
+
+Since the BIP-352 comes with some new primitives (term) (spend key, scan key) & processes (like labels, backing up files), the setup process might be a good opportunity to introduce some of these as needed without overloading the user.
+
+{% include image-gallery.html pages = page.images_sp-onboarding %}
+
+One thing to keep in mind: since silent payments impacts UX flows around sending, syncing & scanning, backup, recovery and applications might even choose to allow users to not adopt the silent payments altogether!
+
+
+## Sending
+
+With the improvements to Contacts, send flows may start from the Contact page in addition to the usual send flow that starts from the app home screen with a Send button. When users start with obtaining a static address, the derived on-chain address is visibly different from the static address the sender started with, something potentially confusing for users. Applications should take measures such as short explainers to avoid confusion on the users part. Applications that do not support silent payment addresses should provide helpful, actionable, human readable errors.
```

Comment:
> send flows may start from the Contact page

We already have this in the contacts page, with the use of lightning addresses.

Reply:
Revised with links to existing page.